bitrise.yml

```diff
@@ -380,12 +380,6 @@ workflows:
         - test_repetition_mode: retry_on_failure
         - maximum_test_repetitions: "5"
         - scheme: PaymentSheet Example
-    - xcode-test@4:
```

Comment:
this intentionally removes financial connections tests from the nightly run

we will have our own runs (even more often than nightly runs), so we don't want other teams to be pinged about our failures